watertap/unit_models/boron_removal.py

```diff
@@ -120,23 +120,34 @@ class BoronRemovalData(InitializationMixin, UnitModelBlockData):
         ),
     )
 
-    # NOTE: This option is temporarily disabled
-    '''
-    CONFIG.declare("energy_balance_type", ConfigValue(
-        default=EnergyBalanceType.useDefault,
-        domain=In(EnergyBalanceType),
-        description="Energy balance construction flag",
-        doc="""Indicates what type of energy balance should be constructed,
-    **default** - EnergyBalanceType.useDefault.
+    CONFIG.declare(
+        "isothermal",
```

Comment:
I think following IDAES conventions this might be better as `is_isothermal`

Reply:
Done in 242f87c.